inspec/a2-api-integration/controls/iam_v2.rb

```diff
@@ -750,4 +750,285 @@
       expect(resp.parsed_response_body[:user_ids]).to eq([user])
     end
   end
+
+  describe "v2beta project rules API" do
+    before(:all) do
+       resp = automate_api_request("/apis/iam/v2beta/projects",
+        http_method: 'POST',
+        request_body: {
+          id: CUSTOM_PROJECT_ID,
+          name: "display name !#$#"
+        }.to_json
+      )
+      expect(resp.http_status).to eq 200
+    end
+
+    after(:all) do
+      resp = automate_api_request("/apis/iam/v2beta/projects/#{CUSTOM_PROJECT_ID}", http_method: 'DELETE')
+      expect(resp.http_status.to_s).to match(/200|404/)
+    end
+
+    context "when the project does not exist" do
```

Comment:
could we rearrange these tests so the project only gets created in a before block for the test cases that require it?

Reply:
I agree that this would be worthwhile.